File: java/src/main/java/model/role/RoleListQueryParam.java
package model.role;


public class RoleListQueryParam {
    private Integer skip;
    private Integer limit;

    private boolean has_internal;
    private boolean has_sync;

    public Integer getSkip() {
        return skip;
    }

    public void setSkip(Integer skip) {
        this.skip = skip;
    }

    public Integer getLimit() {
        return limit;
    }

    public void setLimit(Integer limit) {
        this.limit = limit;
    }

    public boolean isHas_internal() {
        return has_internal;
    }

    public void setHas_internal(boolean has_internal) {
        this.has_internal = has_internal;
    }

    public boolean isHas_sync() {
        return has_sync;
    }

    public void setHas_sync(boolean has_sync) {
        this.has_sync = has_sync;
    }

    @Override
    public String toString() {
        return "RoleListQueryParam{" +
                "skip=" + skip +
                ", limit=" + limit +
                ", has_internal=" + has_internal +
                ", has_sync=" + has_sync +
                '}';
    }
}

File: python/src/model/role/role_list_query_param.py
class RoleListQueryParam:
    def __init__(self, skip, limit):
        self.skip = skip
        self.limit = limit

    def setHsdDync(self, has_sync):
        # int 类型 选填
        self.has_sync = has_sync

    def setHasInternal(self, has_internal):
        # int 类型 选填
        self.has_internal = has_internal
